cinema/serializers.py

```diff
@@ -1 +1,62 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import CinemaHall, Genre, Actor, Movie, MovieSession
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["name", "rows", "seats_in_row", "capacity"]
```

Comment:
The `capacity` field in `CinemaHallSerializer` is included in the `fields` list but is not explicitly defined in the serializer. Since `capacity` is a property in the `CinemaHall` model, it should be added as a `SerializerMethodField` or similar to ensure it's properly serialized. Consider adding a method to handle this field.